src/components/spaces/index/index.js

```diff
@@ -1,12 +1,51 @@
 import React, {Component} from 'react'
 import {connect} from 'react-redux'
+import Icon from 'react-fa'
 
+import DropDown from 'gComponents/utility/drop-down/index'
+import {CardListElement} from 'gComponents/utility/card/index.js'
 import SpaceCards from 'gComponents/spaces/cards'
 
 import * as search from 'gModules/search_spaces/actions'
 
 import './style.css'
 
+const TIMEFRAME_ALL_TIME = 'ALL_TIME'
+const TIMEFRAME_MONTHLY = 'MONTHLY'
+const SORT_BY_POPULAR = 'POPULAR'
+const SORT_BY_RECENT = 'RECENT'
+
+const Filters = ({sortBy, timeframe, onChangeSortBy, onChangeTimeFrame}) => {
+  let sortNames = {}
```

Comment:
I would just declare these inline; when I read a 'let' declaration I presume the variable is going to dynamically change later.